lib/UserInterface/textinput/Callbacks.h

```diff
@@ -30,7 +30,7 @@ namespace textinput {
                           EditorRange& R /*out*/,
                           std::vector<std::string>& DisplayCompletions /*out*/)
     = 0;
-    virtual ~TabCompletion();
+    virtual ~TabCompletion() {}
```

Comment:
I also have a hunch it will fix PowerPC exceptions failing:
https://github.com/root-mirror/cling/commit/494733cef80998a12c28a8f1475dfcb7cc250dd0